src/avfoundation.cs

```diff
@@ -254,6 +255,113 @@ interface AVMutableDateRangeMetadataGroup
 		AVMetadataItem[] Items { get; set; }
 	}
 	
+	[TV (11,0), NoWatch, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVDepthData
+	{
+		[Static]
+		[Export ("depthDataFromDictionaryRepresentation:error:")]
+		[return: NullAllowed]
+		AVDepthData Create (NSDictionary imageSourceAuxDataInfoDictionary, [NullAllowed] out NSError outError);
+
+		[Export ("depthDataByConvertingToDepthDataType:")]
+		AVDepthData Create (uint depthDataType);
+
+		[Export ("depthDataByApplyingExifOrientation:")]
+		AVDepthData Create (CGImagePropertyOrientation exifOrientation);
+
+		[Export ("depthDataByReplacingDepthDataMapWithPixelBuffer:error:")]
+		[return: NullAllowed]
+		AVDepthData CreateD (CVPixelBuffer pixelBuffer, [NullAllowed] out NSError outError);
+
+		[Export ("availableDepthDataTypes")]
+		NSNumber[] AvailableDepthDataTypes { get; }
```

Comment:
candidate for something better than `NSNumber[]` with `[BindAs]`
even if it's `uint[]` ;-)